### PR TITLE
[PLAYER-4007] Added logic for interactions of playback speed logic with RN and core SDK

### PR DESCRIPTION
**Pull request by MarkEPAM**
- added bridge methods for interactions with RN and core SDK
Unit tests not included.
Task: https://jira.corp.ooyala.com/browse/PLAYER-4007

### CODE REVIEW

File: sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m [hunk 1]
```diff
@@ -398,6 +400,22 @@ - (void)bridgeAudioTrackChangedNotification:(NSNotification *)notification {
   [self.viewController sendBridgeEventWithName:notification.name body:eventBody];
 }
 
+- (void)bridgePlaybackSpeedEnabledNotification:(NSNotification *)notification {
+  NSMutableDictionary *eventBody = [NSMutableDictionary new];
+
+  eventBody[@"playbackSpeedEnabled"] = @(self.player.isPlaybackSpeedEnabled);
+  eventBody[@"playbackSpeedRates"] = self.player.availablePlaybackSpeedRates;
```
Comment (pilievOoyala) [inline sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m]: do we need the @ symbol here? You have it for the other two.

Comment (pilievOoyala) [reply sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m]: Is this one a number as opposed to the one below being a string?

**Comment by pilievOoyala** [comment]
LGTM, just one comment but I'm not sure it's needed.

File: sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m (hunk 1 above)
Comment (MarkEPAM) [reply sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m]: @pilievOoyala It's a short syntax for ```NSNumber``` objects. Here it's a converting ```BOOL``` to a ```NSNumber```.  So, below it's an array.

**Review by maxkupetskii — APPROVED** [review]

**Review by brol1dev — APPROVED** [review]
LGTM

**Pull request merged**